demo/EXPLOR3R/auto-drive.py

```diff
@@ -64,9 +66,9 @@ def backup():
     Back away from an obstacle.
     """
 
-    # Sound backup alarm.
+    # Sound backup alarm. 
     Sound.tone([(1000, 500, 500)] * 3)
-
+    
```

Comment:
Is this whitespace consistent with what we have in other places? While we're here, we should normalize this.